homeassistant/components/vesync/common.py

```diff
@@ -0,0 +1,64 @@
+"""Common utilities for VeSync Component."""
+import logging
+
+_LOGGER = logging.getLogger(__name__)
+
+DOMAIN = 'vesync'
```

Comment:
If we need the a constant in multiple modules, please move the constant to a module const.py in the package and import the constant where it's needed.